### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by MaeIsBad**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,5 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   If you're upgrading to opentelemetry_absinthe 1.1.0, it is therefore recommended to also upgrade to OpenTelemetry API 1.1.0
   in order to keep the opentelemetry log metadata.
 
-[Unreleased]: https://github.com/primait/opentelemetry_absinthe/compare/1.1.0...HEAD
+[Unreleased]: https://github.com/primait/opentelemetry_absinthe/compare/1.2.0...HEAD
+[1.2.0]: https://github.com/primait/opentelemetry_absinthe/compare/1.1.0...1.2.0
```
Comment (cpiemontese) [inline CHANGELOG.md:42]: This should be `2.0.0` right?

Comment (MaeIsBad) [reply CHANGELOG.md:42]: yep

**Review by cpiemontese — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,5 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   If you're upgrading to opentelemetry_absinthe 1.1.0, it is therefore recommended to also upgrade to OpenTelemetry API 1.1.0
   in order to keep the opentelemetry log metadata.
 
-[Unreleased]: https://github.com/primait/opentelemetry_absinthe/compare/1.1.0...HEAD
+[Unreleased]: https://github.com/primait/opentelemetry_absinthe/compare/2.0.0...HEAD
+[2.0.0]: https://github.com/primait/opentelemetry_absinthe/compare/1.1.0...2.0.0
+[1.1.0]: https://github.com/primait/opentelemetry_absinthe/releases/tag/1.1.0
```
Comment (cpiemontese) [inline CHANGELOG.md:43]: this is duplicate, just noticed

Comment (MaeIsBad) [reply CHANGELOG.md:43]: :facepalm:

**Review by cpiemontese — APPROVED** [review]

**Pull request merged**